network/load-balancer/how-to/setup-ssl-offloading.mdx

```diff
@@ -83,41 +85,57 @@ Scaleway Load Balancers also support third party SSL Certificates. These are cer
-		- **Proxy**: Enable this option, to use the `Proxyv2` protocol.
-		- **Sticky Session**: Enables the Load Balancer to bind a user’s session to a specific instance. This ensures that all subsequent sessions from the user are sent to the same instance, while there is at least one active session.
-		- **Health Check**: Configure a `HTTP` health check to detect if the backend application is available.
-		- **Server IP(s)**: Enter the IP address of the server(s) running the backend application.
-
-6. Click **Add Frontend** to setup the new frontend and backend.
-
-5. Open a web browser and point it to https://common_name. Make sure to replace `common_name` with the main domain configured in the SSL certificate. The connection is now encrypted with SSL.
+4. Click **Add Frontend** to create a new frontend, or click the <Icon name="more" /> icon next to an existing frontend to edit it.
+
+5. Configure the frontend as follows:
+	- **Frontend name**: A friendly name for the frontend
+	- **Port**: The port number on which the frontend listens. Enter `443` to configure the Load Balancer to listen on the standard `HTTPS` port.
+	- **SSL Certificate**: Choose the **SSL Certificate** to use from the drop-down list.
+
+6. If you are editing an existing frontend:
+	- Save the configuration by clicking **Edit frontend**.
+	- Click the **Backends** tab.
+	- Click the <Icon name="more" /> icon next to the backend to edit it.
+
+	If you are creating a new frontend and backend, continue to the backend creation steps.
+
+7. Configure the backend as follows:
+	- **Backend name**: A friendly name for the backend
+	- **Protocol**: The protocol to use for the backend. Choose `HTTP` from the drop-down list.
+	- **Port**: The port on which the backend application listens. Enter `80` to configure the Load Balancer to communicate with the backend on the standard `HTTP` port.
+	- **Proxy**: Select the `Proxyv2` protocol.
+	- **Health Check**: Configure a `HTTP` health check to detect if the backend application is available.
+	- **Server IP(s)**: Enter the IP address of the server(s) running the backend application.
+    - Configure **advanced settings** if required:
+      - **S3 failover**: activate the feature and enter an Scaleway Object Storage URL. If none of your backend servers are available, users will be redirected to this page.
+      - **Sticky session**: activate the feature and enter the name for a cookie that will bind users' sessions to specific instances. 
+
+8. Click the **Edit** or **Create** button as applicable to finish the process.
+
+9. Open a web browser and point it to https://common_name. Make sure to replace `common_name` with the main domain configured in the SSL certificate. The connection is now encrypted with SSL.
+
+## How to configure SSL bridging
+
+To configure SSL bridging, use TCP protocol on port 443.
```

Comment:
Ideally we need to give more info here (this has been copied/pasted from the console)
Awaiting info from LB team